progress/README.md

```diff
@@ -0,0 +1,256 @@
+				"planned": "make frontend changes for the feature a23",
+				"blockers": "Pending review",
+				"date": "2023-04-30",
+				"createdAt": "2023-04-30T12:00:00Z"
+			}
+		]
+	}
+
+	```
+
+
+
+## **POST /progress**
+
+	Creates a new progress entry.
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**  
+  Attributes:
+    -type (required, string): Specifies the type of the progress entry (e.g., "task", "user").
+    -userId (required, string): User ID associated with the progress entry.
+    -taskId (optional, string): Task ID associated with the progress entry (applicable for task progress only).
+    -completed (required, string): Completed portion of the task or progress.
+    -planned (required, string): Planned portion of the task or progress.
+    -blockers (required, string): Blockers for the progress entry.
+    -date (required, timestamp): Date of the progress entry (applicable for user progress only).
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:** 
+    ```json
+		{
+			"message": "Progress entry created successfully.",
+			"progress": { "<progress_object>" }
+		}
+    ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:** 
+		```json
+		{
+			"statusCode": 401,
+			"error": "Unauthorized",
+			"message": "Authentication credentials are missing or invalid."
+		}
+		```
+  - **Code:** 404
+    - **Content:** 
+  	```json
+		{
+			"statusCode": 404,
+			"error": "Not Found",
+			"message": "Task or user not found"
+		}
+		```
+
+  - **Code:** 500
+    - **Content:** 
+    ```json
+		{
+			"statusCode": 500,
+			"error": "Internal Server Error",
+			"message": "The User Status could not be found as an internal server error occurred."
+		}
+    ```
+- **Example:**
+	POST /progress
+	Content-Type: application/json
+	Request-Body: 
+	```json
+	{
+		"type": "task",
+		"userId": "5d4c3f6079a8b775451db867",
+		"taskId": "6f53574d684dacd0b791773s",
+		"completed": "50%",
+		"planned": "80%",
+		"blockers": "Waiting for approval"
+	}
+	```
+	Response:
+	Status 201 Created
+	Content-Type: application/json
+	```json
+	{
+		"message": "Progress entry created successfully.",
+		"progress": {
+			"id": "5d4c3f6079a8b775451db867",
+			"type": "task",
+			"userId": "d8d7014sd69f535637c47ab7",
+			"taskId": "167cfb7dd45ad967s3380547",
+			"completed": "50%",
+			"planned": "80%",
+			"blockers": "Waiting for approval",
+			"createdAt": 1654101900000
+		}
+	}
+	```
+
+## **DELETE /progress/:id**
+
+Deletes a progress entry with the specified ID. This route is restricted to super users only.
+
+- **Params**  
+  _Required:_ `id=[string]`
+- **Query**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Body**
+  None
+- **Success Response:**
+  - **Code:** 200
+    - **Content:** 
+			```json
+			{
+				"message": "Progress entry with ID '<id>' deleted successfully."
+			}
+			```
```

Comment:
Please fix the output
<img width="761" alt="image" src="https://user-images.githubusercontent.com/46373689/235417817-7a48e55b-a225-446e-968f-95ddb21e5cb2.png">